sdk/storage_datalake/examples/data_lake_04_directory.rs

```diff
@@ -0,0 +1,79 @@
+use azure_storage::storage_shared_key_credential::StorageSharedKeyCredential;
```

Comment:
As a nit: all these example files can probably be renamed from "data_lake_N_$thing.rs" to just "$thing.rs" since it's obvious that they are data lack related and the numbering doesn't really help.

Reply:
This makes sense now that data_lake is a separate crate. The numbering is used in other places of this SDK (inconsistently), but I agree it's better to not have it.

Reply:
done